Src/BootCamp.Chapter/Credentials.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class Credentials
```

Comment:
This is not credentials. This is ``CredentialsManager`` or ``CredentialsRegistry``

Reply:
I updated my code in latest commit [2a4af80](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/232/commits/2a4af80f11d86f38c8efd0a1debe8b0fd5ae911e)